saleor/graphql/order/types.py

```diff
@@ -1178,6 +1183,7 @@ class Order(ModelObjectType[models.Order]):
         deprecation_reason=(f"{DEPRECATED_IN_3X_FIELD} Use `id` instead."),
     )
     voucher = graphene.Field(Voucher)
+    voucher_code = graphene.Field(VoucherCode)
```

Comment:
I'm not sure if this should be `VoucherCode` field. I mean IMO we want to return `voucher_code` even if the voucher code instance does not exist anymore.

@fowczarek WDYT?

Reply:
now voucher_code is a String